<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisionMainWindow</class>
 <widget class="QMainWindow" name="VisionMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisionMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="left_disp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_disp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openVisonbtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>打开视觉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadDllBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>加载策略</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DllPathDisp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuconctSet">
    <property name="title">
     <string>连接设置</string>
    </property>
    <addaction name="actionSeriportSet"/>
    <addaction name="actionTcpSet"/>
   </widget>
   <widget class="QMenu" name="menuCamSet">
    <property name="title">
     <string>相机设置</string>
    </property>
    <addaction name="actionCamDevSet"/>
    <addaction name="actionImageSet"/>
   </widget>
   <addaction name="menuconctSet"/>
   <addaction name="menuCamSet"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSeriportSet">
   <property name="text">
    <string>串口设置</string>
   </property>
  </action>
  <action name="actionTcpSet">
   <property name="text">
    <string>服务器设置</string>
   </property>
  </action>
  <action name="actionCamDevSet">
   <property name="text">
    <string>相机设备设置</string>
   </property>
  </action>
  <action name="actionImageSet">
   <property name="text">
    <string>图像预处理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
